e2e/support/helpers/index.js

```diff
@@ -36,3 +36,4 @@ export * from "./e2e-models-metadata-helpers";
 export * from "./e2e-ui-elements-overflow-helpers";
 export * from "./e2e-relative-date-picker-helpers";
 export * from "./e2e-command-palette-helpers";
+export * from "./e2e-table-metadata-helpers";
```

Comment:
```suggestion
```

I think the rest of E2E tests intentionally used `import { createMetric } from "e2e/support/helpers/e2e-table-metadata-helpers";` instead of re-exporting table metadata helpers here.

`createSegment` and `createMetric` are likely to be deprecated in the near future so their use is discouraged.

Reply:
both variants will be easy to drop, I don't follow why one is better than another

Reply:
I don't have a strong opinion.
Just noticed that other tests explicitly import it from the separate folder. One thing that comes to mind as to why that approach might be "better" is for easier "searchability" later one when we decide to get rid of these helpers for good.

Anyway, doesn't bother me either way since we didn't explicitly deprecate anything and we don't have a lint rule that would prevent this.

Reply:
I think there is no right or wrong answer here so both options are acceptable and should be non blocking